docs/repositories/files.md

```diff
@@ -59,9 +59,9 @@ The Files page only displays files in your repository that were analyzed by Coda
 
     Not all files may exist in all branches of your repositories. Make sure that you're displaying files for the correct branch.
 
--   **The file might have been ignored**
+-   **The file might be ignored**
 
-    The Files page doesn't display [ignored files](../repositories-configure/ignoring-files.md) that aren't meant to be analyzed.
+    The Files page doesn't display [ignored files](../repositories-configure/ignoring-files.md) that aren't meant to be analyzed, including the [files that Codacy ignores by default](../repositories-configure/ignoring-files.md#default-ignored-files).
```

Comment:
@madalenamelo, instead of following your suggestion to include the list of files ignored by default on this page, I'm linking to a section on the page about ignoring files that already included this information. This is to avoid repeating the same information in more than one place, but hopefully, it's enough to call the attention to users who may run into this scenario.